Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityPointer.cs

```diff
@@ -57,6 +57,11 @@ public interface IMixedRealityPointer : IEqualityComparer
         /// </summary>
         bool IsFocusLocked { get; set; }
 
+        /// <summary>
+        /// Specifies whether the pointer's target position (cursor) is locked as well when focus is locked.
```

Comment:
>locked [](start = 72, length = 6)

Maybe add locked 'to the target'? I didn't realize what it meant on first read.

Reply:
focus lock already means locked to the target. This is specifically for the target position. So "locked to the position target on the target"?